test/ttsafe_attr_vlen.c

```diff
@@ -6,7 +6,7 @@
  * This file is part of HDF5.  The full HDF5 copyright notice, including     *
  * terms governing use, modification, and redistribution, is contained in    *
  * the COPYING file, which can be found at the root of the source code       *
- * distribution tree, or in https://www.hdfgroup.org/licenses.               *
+ * distribution tree, or in https://support.hdfgroup.org/ftp/HDF5/releases.  *
```

Comment:
Should I change this back? I just copied the entire file over from develop.

Reply:
Yes, it should be the line ending in "licenses."  I'll be updating develop in the near future.

Reply:
Fixed